diff --git a/configure.ac b/configure.ac
index e23a2ed..8ccce25 100644
--- a/configure.ac
+++ b/configure.ac
@@ -86,7 +86,7 @@ AC_SUBST(LFLAGS)
 CFLAGS="$CFLAGS -Wall"
 LOCALE_T=locale_t
 AS_CASE([$host],
-        [*mingw*], [LDFLAGS="$LDFLAGS -no-undefined" CFLAGS="$CFLAGS -D_spawnv=spawnv"], [])
+        [*mingw*], [LDFLAGS="$LDFLAGS -no-undefined" LOCALE_T=_locale_t], [])
 AC_SUBST(LOCALE_T)
 
 dnl See if iconv is present and wanted
